tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -1070,25 +1078,34 @@ describe(__filename, () => {
         );
       });
 
-      it('should dispatch a fetch action if page changes', () => {
+      it('should dispatch a fetch action if page changes', async () => {
         const dispatch = jest.spyOn(store, 'dispatch');
         signInUserAndRenderUserProfile();
         loadAddonsByAuthors({
           count: EXTENSIONS_BY_AUTHORS_PAGE_SIZE + 1,
           store,
         });
 
-        userEvent.click(screen.getByRole('link', { name: 'Next' }));
+        let link;
 
-        expect(dispatch).toHaveBeenCalledWith(
-          fetchAddonsByAuthors({
-            addonType: ADDON_TYPE_EXTENSION,
-            authorIds: [defaultUserId],
-            errorHandlerId: createErrorHandlerId(),
-            page: '2',
-            pageSize: String(EXTENSIONS_BY_AUTHORS_PAGE_SIZE),
-            sort: SEARCH_SORT_POPULAR,
-          }),
+        await waitFor(() => {
+          link = screen.getByRole('link', { name: 'Next' });
```

Comment:
Could this be changed to `link = await screen.findByRole('link', { name: 'Next' });` and avoid the `waitFor`?

Reply:
Probably - looks like because it's assigned to a variable the `prefer-find-by` lint rule didn't catch this. I'll take a look.